src/renderer/assets/styles/index.css

```diff
@@ -15,6 +15,7 @@
   --themeColor30: rgba(33, 181, 111, .3);
   --themeColor20: rgba(33, 181, 111, .2);
   --themeColor10: rgba(33, 181, 111, .1);
+  --highlightThemeColor: var(--themeColor);
```

Comment:
I think no need to declare a new variable if they have the same value.

Reply:
I have not changed anything on Element UI but there are more wrong colors. If you mean the left down arrow background and color that's because it's disabled (2 is the minimum value) but this is the same as in v0.14.0.